WordPress/src/main/java/org/wordpress/android/ui/accounts/applicationpassword/ApplicationPasswordLoginActivity.kt

```diff
@@ -29,12 +31,27 @@ class ApplicationPasswordLoginActivity: BaseAppCompatActivity() {
         viewModel!!.setupSite(intent.dataString.orEmpty())
     }
 
-    private fun runMainIdNecessary(credentialsStored: Boolean) {
-        if (credentialsStored) {
+    private fun runMainIdNecessary(sireUrl: String?) {
```

Comment:
I mentioned this on a previous PR: this function name doesn't describe to me what the function does. Perhaps it was meant to be named `runMainIfNecessary` instead of `runMainIdNecessary` (so **If** instead of **Id**)?